solver/src/liquidity.rs

```diff
@@ -0,0 +1,75 @@
+use anyhow::Result;
+use model::order::OrderKind;
+use num_rational::Rational;
+use primitive_types::{H160, U256};
+use settlement::{Interaction, Trade};
+use std::sync::Arc;
+
+#[cfg(test)]
+use mockall::automock;
+
+use crate::settlement;
+
+///
+/// Defines the different types of liquidity our solvers support
+///
```

Comment:
Why surround doc comment with empty lines? Not important but I prefer just having the comment.